lib/repo/git_repository.rb

```diff
@@ -1088,34 +1092,6 @@ def last_modified_date()
 
     private
 
-    def files_at_path_helper(path = '/', only_changed = false)
-      if path.nil?
-        path = '/'
-      end
-      result = Hash.new(nil)
-      raw_contents = self.__get_files(path, @revision_number)
-      raw_contents.each do |file_name, type|
-        if type == :file
-          last_modified_date = @repo.__get_node_last_modified_date(File.join(path, file_name), @revision_number)
-          last_modified_revision = @repo.__get_history(File.join(path, file_name), nil, @revision_number).last
-
-          if(!only_changed || (last_modified_revision == @revision_number))
-            new_file = Repository::RevisionFile.new(@revision_number, {
-                                                      :name => file_name,
-                                                      :path => path,
-                                                      :last_modified_revision => last_modified_revision,
-                                                      :changed => (last_modified_revision == @revision_number),
```

Comment:
I'm not sure whether this line is correct, but it looks like it was the only place that "changed" could be something other than true.